rule/platform.go

```diff
@@ -133,6 +150,8 @@ func init() {
 		KnownOSArchs[p.OS] = append(KnownOSArchs[p.OS], p.Arch)
 		KnownArchOSs[p.Arch] = append(KnownArchOSs[p.Arch], p.OS)
 	}
+	// unix is a pseudo os
+	KnownOSSet["unix"] = true
```

Comment:
This doesn't feel great, but neither does adding unix+arches to KnownPlatforms... unix is an alias to other oses, but the various oses have different arch support. Not sure the correct/expected behavior here.

Reply:
This caused other implications. Changed in 276dbe9